.cirrus.yml

```diff
@@ -1,11 +1,9 @@
 libcxxrt_freebsd_task:
   matrix:
-  - freebsd_instance:
```

Comment:
This wants updating to 13.5, not removing.

Reply:
Ah sorry misread https://cirrus-ci.org/guide/FreeBSD/#list-of-available-image-families and thought only 14.x and 15.x are available.